modules/ocf_apphost/manifests/proxy.pp

```diff
@@ -16,15 +16,32 @@
       require => Package['nginx'],
       notify  => Service['nginx'];
 
-    '/usr/local/sbin/rebuild-vhosts':
-      source  => 'puppet:///modules/ocf_apphost/rebuild-vhosts',
+    '/usr/local/sbin/build-vhosts':
+      source  => 'puppet:///modules/ocf_www/build-vhosts',
+      mode    => '0755';
+
+    '/opt/share/vhost-app.jinja':
+      source  => 'puppet:///modules/ocf_apphost/vhost-app.jinja';
+
+    # Generated SSL bundles go here
+    '/etc/ssl/apphost':
+      ensure  => directory,
       mode    => '0755';
   }
 
-  cron {
-    'rebuild-vhosts':
-      command => '/usr/local/sbin/rebuild-vhosts > /dev/null',
-      minute  => '*/10',
-      require => Package['nginx'];
+  if $dev_config {
+    cron {
+      'build-vhosts':
+        command => '/usr/local/sbin/build-vhosts --dev app > /dev/null',
```

Comment:
i'd suggest using chronic instead of always piping to /dev/null

this way we get the output on failures, but otherwise it gets sent to /dev/null